io.github.KangLin.SerialPortAssistant.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "io.github.KangLin.SerialPortAssistant",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "SerialPortAssistant",
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=all",
+        "--filesystem=xdg-documents",
+        "--talk-name=org.freedesktop.Notifications",
```

Comment:
notifications does not go through the portal? What do you use for notifications?